salesforce/10.14/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -4037,14 +4037,24 @@ Updates one or more metadata components in your organization.
 
 Upserts a homogeneous list of objects. Creates new records and updates existing records using a custom field to determine the presence of existing records. In most cases, use `#upsert(String, String, List, Map)` over `#create(String, List, Map)` to avoid creating unwanted, duplicate records.
 
+[NOTE]
+The *Upsert* operation does not work with the SObject `priceBookentry2`.
+
+[NOTE]
+Although you cannot change the *Content Type* value for a bulk upsert, you can use the <<createJob, *Create Job*>> operation to set the content type to CSV (or zipped CSV if you reach the character limit) and then follow up with the <<createBatch, *Create Batch*>> operation.
 
 
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | Name of the configuration to use. | | x
 | Object Type a| String |  Type of record to upsert. |  | x
-| External Id Field Name a| String |  Contains the name of the field on this object with the external ID field attribute for custom objects or the ID lookup field property for standard objects. |  | x
+| External Id Field Name a| String |  Contains the name of the field on this object with the external ID field attribute for custom objects or the ID lookup field property for standard objects.
```

Comment:
I find this confusing. I suggest something like this:

Either of the following:

* For a custom object, the name of the field that contains the external ID field attribute (should this be field name instead of attribute?)
* For a standard object, contains the name of the field that contains the ID lookup field name